Dockerfile

```diff
@@ -15,6 +15,5 @@ RUN chmod +x /bin/docker
 
 COPY . /usr/src/app
 
-ENV FILESYSTEM_DIR /code
-
+ENV CODECLIMATE_DOCKER 1
```

Comment:
Bleh, you're right, I didn't think it through all the way. Yeah, this seems like the less troubling option.